src/html/partials/key-factors.html

```diff
@@ -27,8 +27,8 @@ <h2 class="sm-title">Why People Don't Adhere</h2>
   <div class="container -lg-width list-wrapper">
 
     <ul class="tag-list">
-      <li class="tag-list__item">
-        <button class="tag-list__btn -gear" aria-label="Show Comprehension Factors" >
+      <li class="tag-list__item test">
+        <button class="tag-list__btn -gear" aria-label="Show Comprehension Factors" aria-pressed="false" data-module="ExpandListItem">
```

Comment:
Instead of `aria-pressed`, I'd suggest `aria-expanded` (true/false) with `aria-controls` (id of the target element).